infrastructure/parallelcluster-ui.yaml

```diff
@@ -209,12 +209,12 @@ Resources:
       Description: 'ANY integration'
       IntegrationType: AWS_PROXY
       IntegrationUri: !Sub
-        - arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/arn:${AWS::Partition}:lambda:${AWS::Region}:${AWS::AccountId}:function:PclusterManagerFunction-${StackIdSuffix}/invocations
+        - arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/arn:${AWS::Partition}:lambda:${AWS::Region}:${AWS::AccountId}:function:ParallelClusterUIFunction-${StackIdSuffix}/invocations
         - { StackIdSuffix: !Select [2, !Split ['/', !Ref 'AWS::StackId']] }
       PayloadFormatVersion: 2.0
       TimeoutInMillis: 30000
 
-  PCMApiGatewayAccessLog:
+  ApiGatewayAccessLog:
```

Comment:
chore: @tmscarla remember to update also the logging retrieval guide if you edit the name of this log group